R/model-module-helpers.R

```diff
@@ -1,10 +1,10 @@
 #' Identify report dates with complete (i.e up to the maximum delay) reference
 #' dates
 #'
-#' @param new_confirm `new_confirm``data.frame`output from
+#' @param new_confirm `new_confirm` `data.frame` output from
```

Comment:
not sure if this is wrong or not. seems fishy.

Reply:
its correct but perhaps not clear `new_confirm` is a `data.frame` output as part of the `epinowcast` output.